test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/ldap/ClusteredWorkbenchKieServerPersistentScenarioLdapIntegrationTest.java

```diff
@@ -52,10 +53,14 @@ public static void initializeDeployment() {
                 .withLdapRoleRecursion(LdapSettingsConstants.ROLE_RECURSION)
                 .withLdapDefaultRole(LdapSettingsConstants.DEFAULT_ROLE).build();
 
+        try {
         deploymentScenario = deploymentScenarioFactory.getClusteredWorkbenchKieServerDatabasePersistentScenarioBuilder()
                 .withLdapSettings(ldapSettings).withExternalMavenRepo(MavenConstants.getMavenRepoUrl(),
                         MavenConstants.getMavenRepoUser(), MavenConstants.getMavenRepoPassword())
                 .build();
+        } catch (UnsupportedOperationException ex) {
+            Assume.assumeFalse(ex.getMessage().startsWith("Not supported"));
```

Comment:
What is the purpose of this catch?

Reply:
If scenario configuration is not possible, then all tests are skipped.